Sources/TensorFlow/Operators/Comparison.swift

```diff
@@ -227,3 +227,16 @@ public extension StringTensor {
         return Raw.equal(self, other)
     }
 }
+
+public extension Tensor where Scalar: FloatingPoint {
+    /// Returns `true` if tensors are of equal shape and all pairs of scalars are approximately
+    /// equal.
+    @inlinable
+    func isAlmostEqual(
+        to other: Tensor,
+        tolerance: Scalar = Scalar.ulpOfOne.squareRoot()
+    ) -> Bool {
+        return self.shape == other.shape &&
```

Comment:
You can use `Tensor.elementsApproximatelyEqual(_:tolerance:)` directly to make this faster. Also, maybe we should also add the `to` argument label to all these other functions that check for element-wise equality.

Reply:
> Also, maybe we should also add the `to` argument label to all these other functions that check for element-wise equality.

This `elementsApproximatelyEqual(_:tolerance:)` method name is a sentence rather than a verb phrase where "equal" is a transitive verb. Same for `Sequence.elementsEqual(_:)`.

We should change "approximately" to "almost" to for consistency with `isAlmostEqual(to:)`,

Reply:
I've renamed `elementsApproximatelyEqual` to `elementsAlmostEqual` and this is now called from `isAlmostEqual`. For consistency I also changed the condition on `Scalar` to now conform to `TensorFlowFloatingPoint` so that it's convertible to `Double`. Ptal.